data-structures/ds-pque/java/PriorityQueue.java

```diff
@@ -0,0 +1,64 @@
+public class PriorityQueue {
+    public static class Node {
+        public int data;
+        public int priority;
+        public Node next;
+
+    }
+
+    Node node = new Node();
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6336404)